docs/Reference/LTS.md

```diff
@@ -47,8 +47,8 @@ A "month" is defined as 30 consecutive days.
 | :------ | :----------- | :-------------- | :------------------- |
 | 1.0.0   | 2018-03-06   | 2019-09-01      | 6, 8, 9, 10, 11      |
 | 2.0.0   | 2019-02-25   | 2021-01-31      | 6, 8, 10, 12, 14     |
-| 3.0.0   | 2020-07-07   | TBD             | 10, 12, 14, 16, 18   |
-| 4.0.0   | TBD          | TBD             | 14, 16, 18           |
+| 3.0.0   | 2020-07-07   | 2022-06-30      | 10, 12, 14, 16, 18   |
```

Comment:
Will `2022-06-30` be too soon? It just 15 days left and I am afraid it will not be enough time for most company to decide the transition of `fastify@3` to `fastify@4`.

Reply:
This is at odds with the v4 blog post, which states 2023-06-30: https://medium.com/@fastifyjs/fastify-v4-ga-59f2103b5f0e

Reply:
good spot, I was off one year.